ui-core/src/components/Select.tsx

```diff
@@ -64,16 +58,14 @@ const Select = <T,>({
           small,
           'read-only': readOnly,
         })}
-        value={selectedOption ? getOptionValue(selectedOption) : undefined}
         required={required}
         disabled={disabled || readOnly}
         onChange={handleOnChange}
       >
-        {placeholder && <option value={PLACEHOLDER_VALUE}>{`– ${placeholder} –`}</option>}
+        {placeholder && <option>{`– ${placeholder} –`}</option>}
         {options.map((option) => (
-          <option key={getOptionValue(option)} value={getOptionValue(option)}>
-            {getOptionLabel(option)}
-          </option>
+          /* eslint-disable-next-line react/jsx-key */
```

Comment:
We loose the `key` here, but since these lists are quite short I don't believe it matters.

Reply:
That’s a generic component, We don’t know who is going to use it, and it could be a select of any size.
Besides, I think even with smaller lists react can get confused and mix up elements between renders if you don’t use keys.